docs/articles/developer-info/Packaging-the-Console-and-Engine.md

```diff
@@ -139,7 +139,7 @@ Packages are archived on nunit.org in the downloads directory. Create a new sub-
 
 ### Github
 
-1. Log onto Github and go to the main nunit repository at <https://github.com/nunit/nunit>.
+1. Log onto Github and go to the main NUnit repository at <https://github.com/nunit/nunit>.
```

Comment:
The name of the repository is actually "nunit" -- maybe should be `nunit` to be clear.

Reply:
I see it as "the repository of NUnit". Since NUnit is the product, I capitalized.

Reply:
Actually, there is no "NUnit" product. There used to be, under V2. But then we spliti it up into multiple products. No need to change, we can standardize this at a later point of we think it's important.

BTW, when version is important, we traditionally say "NUnit V2" versus "NUnit 3" - it's just history.